packages/train2go-bridge/test/parser.test.js

```
@@ -165,6 +165,45 @@ describe("parser", () => {
       expect(activities[0].description).toContain("3x15' Z3 d/5' Z1");
     });
 
+    it("should preserve <li> bullet line breaks in description (regression: <ul><li> concatenation)", () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)